frontend/src/api/.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
```

Comment:
You can remove this file. We are using Github actions. If you're up for setting up CI for these tests, you can create a workflow like [this one](https://github.com/codeforboston/jobhopper/blob/11d547a9c36681fdd94109d13d275513f9764c39/.github%2Fworkflows%2Flint-test.yml)